lunatrace/bsl/frontend/src/components/finding/VulnerabilityTableItem.tsx

```diff
@@ -11,15 +11,16 @@
  * limitations under the License.
```

Comment:
The finding component is in the "project" directory because it is a single use component that only lives in this page.  If you think the page is unclear, we could break the page out of project and have it in its own "snapshot/build" folder, since it's a little deep.  Components are for reusable components like buttons, headers, utility components, etc.

Finding will probably stay attached to the snapshot since this a very snapshot-centric view.